services/src/api/model/operators.rs

```diff
@@ -98,3 +102,352 @@ pub enum TypedResultDescriptor {
     Raster(RasterResultDescriptor),
     Vector(VectorResultDescriptor),
 }
+
+#[derive(PartialEq, Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct MockDatasetDataSourceLoadingInfo {
+    pub points: Vec<Coordinate2D>,
+}
+
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct StaticMetaData<L, R, Q> {
```

Comment:
you have to make sure to use only the types in services/api/model in the handlers. Right now you still use the ones from the operators crate. Please use the ones from here instead and implement the From trait to convert the API types into the ones from the operators crate used in the engine.

Reply:
a2f68fd TypedResultDescriptor & co
7bedf74 MetaData, Geometry & co